projects/igniteui-angular/src/lib/slider/slider.component.ts

```diff
@@ -1054,6 +1057,11 @@ export class IgxSliderComponent implements
                 takeUntil(this._destroyer$)).subscribe(() => this._ngZone.run(() => {
                     this.stepDistance = this.calculateStepDistance();
                 }));
+            fromEvent(this._el.nativeElement, 'pointermove').pipe(
+                throttle(() => interval(0, animationFrameScheduler)),
+                takeUntil(this._destroyer$)).subscribe(($event: PointerEvent) => this._ngZone.run(() => {
```

Comment:
Not sure why `zone.run` would be required, the event is not subscribed is `runOutsideAngular` or anything.

Reply:
A few lines above you can see that it is ran outside angular actually.

Reply:
Ah lol didn't expand the diff up enough to see it